3-platform/shopping-cart/infra/src/main/java/com/_6core/platform/shopping/cart/infra/adapter/driving/grpc/CreateOrderGrpc.java

```diff
@@ -0,0 +1,9 @@
+package com._6core.platform.shopping.cart.infra.adapter.driving.grpc;
```

Comment:
move to domain, on the infra should be implementation
```suggestion
package com._6core.platform.shopping.cart.infra.adapter.driving.grpc;
```

How do you plan to use this interface from here?